src/schoology-extractor/tests/api/test_request_client.py

```diff
@@ -32,247 +31,276 @@ def test_given_custom_url_then_use_it_for_the_base_url(self):
             custom_url = "a_custom_url"
 
             # Act
-            request_client = RequestClient(
-                FAKE_KEY, FAKE_SECRET, custom_url)
+            request_client = RequestClient(FAKE_KEY, FAKE_SECRET, custom_url)
 
             # Assert
             assert request_client.base_url == custom_url
 
-        def test_given_None_as_parameters_then_assert_should_fail(self):
-            # Assert
-            with pytest.raises(AssertionError):
-                RequestClient(None, None)   # type: ignore
-
     class Test_when_building_request_header:
         def test_then_consumer_key_is_present(self, default_request_client):
             assert self._present_in_header(
                 default_request_client._request_header,
                 header_key="Authorization",
-                contained_value=FAKE_KEY
+                contained_value=FAKE_KEY,
             )
 
         def test_then_oauth_word_is_present(self, default_request_client):
             assert self._present_in_header(
                 default_request_client._request_header,
                 header_key="Authorization",
-                contained_value="OAuth"
+                contained_value="OAuth",
             )
 
         def _present_in_header(self, request_header, header_key, contained_value):
```

Comment:
I'd put this kind of functions outside of the class.

Reply:
You mean this "private" method? If it isn't needed anywhere else, why put it outside the class?

Reply:
I was thinking about leaving it outside of the class because of it does not depend on the class nor helps with readability for the test, but yes, if it is only used here, makes sense to have it here.